front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/ItineraryMarkersV2.tsx

```diff
@@ -78,11 +78,20 @@ const extractMarkerInformation = (pathSteps: (PathStep | null)[]) =>
     return acc;
   }, [] as MarkerInformation[]);
 
-const ItineraryMarkers = ({ map }: { map: Map }) => {
+const ItineraryMarkers = ({
+  map,
+  simulationPathSteps,
+}: {
+  map: Map;
+  simulationPathSteps?: PathStep[];
+}) => {
   const { getPathSteps } = useOsrdConfSelectors();
   const pathSteps = useSelector(getPathSteps);
 
-  const markersInformation = useMemo(() => extractMarkerInformation(pathSteps), [pathSteps]);
+  const markersInformation = useMemo(
+    () => extractMarkerInformation(simulationPathSteps || pathSteps),
```

Comment:
Not sure to understand why we might need `simulationPathSteps` ? We don't pass it to the map from StdcmConfig but informations seem to be the same as in StdcmResults.

Reply:
After discussions, we need it in case we change some inputs in the config so it doesn't affect the markers in the results map